src/components/blocks/blocks.html

```diff
@@ -59,8 +56,8 @@ <h1>Blocks <small data-ng-if="vm.blocks.length > 1">{{vm.blocks[0].height}} &#85
                                 <a class="hidden-xs" href="/address/{{b.generator}}">{{b.delegate.username || b.generator}}</a>
                                 <a class="visible-xs" href="/address/{{b.generator}}">{{b.delegate.username || b.generator | middleEllipsis:10}}</a>
                             </td>
-                            <td class="text-right hidden-xs">{{b.totalAmount | currency:$root.currency:$root.decimalPlaces}}</td>
-                            <td class="right-padding-m right-padding-l double text-right">{{b.totalForged | currency:$root.currency}}</td>
+                            <td class="text-right hidden-xs">{{b.totalAmount | currency:$root.currency:$root.decimalPlaces}} {{$root.currency.symbol}}</td>
+                            <td class="right-padding-m right-padding-l double text-right">{{b.totalForged | currency:$root.currency:1}} {{$root.currency.symbol}}</td>
```

Comment:
Using only one decimal place here will lead to 0.0 when BTC is selected as currency (which is wrong).